### ISSUE TITLE
E8372h-320 (not working) vs E8372h-153 (working)

### ISSUE THREAD

**Issue by kernelport** [pablo/huawei-modem-python-api-client#22]
Hi,
I work with both modems. The new Revision E8372h-320 is able to connect with ipv6 and ipv4.
But sending SMS produce this error:
{'type': 'error', 'error': {'code': '100005', 'message': ''}}
With the E8372h-153 I get:
{'type': 'response', 'response': 'OK'}
Sending SMS ist possible over the webinterface. 
Where is the error code 100005 documented?
Regards,
 Frank

**Comment by pablo** [comment]
I see. I have no idea what's the `100005` error. There is no official API on the MODEMS. 

Could you give me remote (temporary) access to your modems so I can take a look at it? You can use "NGROK" to do that.

**Comment by v1ctor** [comment]
@pablo @kernelport
Apparently according to this [php client](https://github.com/HSPDev/Huawei-E5180-API/blob/bcc2da7886279b415e17ae89025292d30c31b768/README.md) error 100005 is `ERROR_FORMAT_ERROR = 100005`.

I've made an experiment and found the problem. Basically as it says something is fishy with the formatting.
How to reproduce:
```sh
▶ python3 sample.py
{'error': {'code': '100005', 'message': ''}, 'type': 'error'}
```
remove whitespaces from the xml_data:
```diff
diff --git a/huaweisms/api/sms.py b/huaweisms/api/sms.py
index b18acd3..1b3e160 100644
--- a/huaweisms/api/sms.py
+++ b/huaweisms/api/sms.py
@@ -55,19 +55,7 @@ def send_sms(ctx, dest, msg):
     dest = [dest] if isinstance(dest, six.string_types) else dest

     phones_content = '\n'.join('<Phone>{}</Phone>'.format(phone) for phone in dest)
-    xml_data = """
-        <request>
-            <Index>-1</Index>
-            <Phones>
-            {}
-            </Phones>
-            <Sca></Sca>
-            <Content>{}</Content>
-            <Length>{}</Length>
-            <Reserved>1</Reserved>
-            <Date>{}</Date>
-        </request>
-    """.format(phones_content, msg, len(msg), now_str)
+    xml_data = """<request><Index>-1</Index><Phones>{}</Phones><Sca></Sca><Content>{}</Content><Length>{}</Length><Reserved>1</Reserved><Date>{}</Date></request>""".format(phones_content, msg, len(msg), now_str)

     headers = {
         '__RequestVerificationToken': ctx.token,
@@ -119,5 +107,3 @@ def sms_set_read(ctx, index):
     }
     url = "{}/sms/set-read".format(ctx.api_base_url)
     return post_to_url(url, xml_data, ctx, headers)
```

```sh
▶ python3 sample.py
{'response': 'OK', 'type': 'response'}
```

**Comment by dopstar** [comment]
nice find @v1ctor ! this is now fixed in version `1.0.11`

```
pip install -U huawei-modem-api-client
```
